MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/examples/location/CustomizeUserLocationActivity.java

```diff
@@ -38,7 +38,6 @@ protected void onCreate(Bundle savedInstanceState) {
 
     // Get the location engine object for later use.
     LocationEngine locationEngine = LocationSource.getLocationEngine(this);
-    locationEngine.activate();
     locationEngine.requestLocationUpdates();
```

Comment:
Yea, works after merging master back in. 

`activate();` and  `requestLocationUpdates();` have both been removed